backends/cfg_html_doc/templates/csr.adoc.erb

```diff
@@ -100,7 +100,7 @@ This field has special behavior when written by software (_e.g._, through `csrrw
 +
 When software tries to write `csr_value`, the field will be written with the return value of the function below.
 +
-<%- if arch_def.multi_xlen? && csr.defined_in_all_bases? && field.defined_in_all_bases? -%>
+<%- if arch_def.multi_mxlen? && csr.defined_in_all_bases? && field.defined_in_all_bases? -%>
```

Comment:
multi_xlen should stay multi_xlen; it is indicating whether or not, in some privilege mode, the xlen can change between 32 and 64 (not M-mode specific)